dff/core/pipeline/conditions.py

```diff
@@ -13,18 +13,23 @@
 def always_start_condition(_: Context, __: Actor) -> bool:
     """
     Condition that always allows service execution, it's the default condition for all services.
-    :ctx: - current dialog context.
-    :actor: - pipeline actor.
     Returns bool (True).
+
+    :param ctx: current dialog context.
+    :type ctx: Context
```

Comment:
Do we need :type somesting: yet?

Reply:
No. I think I forgot to delete this line.